### ISSUE TITLE
Add "¿?" as a symbol for Spanish subtype

### ISSUE THREAD

**Issue by another-sapiens** [florisboard/florisboard#659]
In Spanish (and probably other languages), when you want to write a question, it starts out with the "¿" symbol. For that, you need to hold "?" until "¿" pops up, and then press "?" again when the question's over. 

A nice feature for grammar-caring writers would be to have an extra "¿?" symbol. Upon tapping on it, the cursor should move backwards so it's between the ¿ and the ?, and the caps should be automatically cativated.  

In simpler terms:

- Tap on "¿?"

- The cursor should be in the middle of the two signs:  ¿+?   ("+" represents the cursor)

- The caps should be automatically activated

Do you think this could be implemented in a future release?

**Comment by kj7rrv** [comment]
Also for ¡!?On Apr 13, 2021 17:09, another-sapiens ***@***.***> wrote:

In Spanish (and probably other languages), when you want to write a question, it starts out with the "¿" symbol. For that, you need to hold "?" until "¿" pops up, and then press "?" again when the question's over.

A nice feature for grammar-caring writers would be to have an extra "¿?" symbol. Upon tapping on it, the cursor should move backwards so it's between the ¿ and the ?, and the caps should be automatically cativated.

In simpler terms:



Tap on "¿?"



The cursor should be in the middle of the two signs:  ¿+?   ("+" represents the cursor)



The caps should be automatically activated



Do you think this could be implemented in a future release?

**Comment by Atreyu-94** [comment]
In Spanish this would be an invaluable time saver.

**Comment by Glitchy-Tozier** [comment]
Sounds like it could be combined with another feature: Easy formatting symbols.
This could do the same thing, but with *|*, _|_, ~|~, or other common symbols.

**Comment by another-sapiens** [comment]
@Glitchy-Tozier 
That's even better, now that I think about it!

**Comment by patrickgold** [comment]
Thanks for your suggestion! This is indeed a very cool idea, never thought about that. Will add it to my todo list, probably for v0.5.0.

As for the symbols: I will definitely add it for "¿?" and "¡!". For the other suggested symbols I will think about it, but these symbols are sometimes used alone and then in pairs, which could be pretty annoying after some time of using it.

**Comment by Temporalin** [comment]
Hi! Just trying Floris Board and not having "¿" easily accessible in a Spanish keyboard is a bit frustrating. I would put it alongside "?" in the "M" submenu, just like "A" has multiple accented "Á" (none of which are actually used in Spanish besides Á)
